### ISSUE TITLE
Extension ruins ASCII art definitions for grid-template-areas

### ISSUE THREAD

**Issue by franktopel** [ronilaukkarinen/vscode-stylefmt#1]
As of now, your extension formats

```css
grid-template-areas:
    "header header header"
    "aside   main    help"
    "footer footer footer";
```

into 

```css
grid-template-areas: "header header header" "aside main help" "footer footer footer";
```

which it really shouldn't do. 

It is common and really desirable to define grid-template-areas in "ASCII art" style. Your formatter shouldn't ruin those; otherwise it becomes kind of unusable if you're working with css grid in a project.

**Comment by rollecode** [comment]
@franktopel Thank you for the first issue and thank you for using vscode-stylefmt. I had no idea someone still uses this except me after [mrmlnc/vscode-stylefmt](https://github.com/mrmlnc/vscode-stylefmt) got archived. This issue seems to exist also in the original repository. 

I myself don't use `grid-template-areas` that much. That's why I have not encountered this issue before. Stylefmt tries to honor [stylelint](https://github.com/stylelint/stylelint) rules but there is no rules for CSS grids AFAIK. This means I need to make and exception to the extension . Will fix soon.

**Comment by rollecode** [comment]
@franktopel This is fixed in 2.9.3.
